cfpb/consumerfinance.gov
# Ask: change date label text

Change the label for the date on Ask answer pages.

## Additions

- `last reviewed` in translations file


## Changes

- Change date label text from `updated` to `last reviewed` on Ask answer pages


## How to test this PR

1. Check out branch
2. Navigate to [an Ask answer page](http://localhost:8000/ask-cfpb/what-effect-will-shopping-for-an-auto-loan-have-on-my-credit-en-763/) and verify that date label is now `last reviewed`
3. Click link to Spanish version of page and verify that date label is now `revisado`


## Screenshots
### English page
<img width="748" alt="Screen Shot 2020-09-24 at 9 45 54 PM" src="https://user-images.githubusercontent.com/778171/94227601-b5bcc280-feaf-11ea-85d8-bc3497a84d1a.png">


### Spanish page
<img width="787" alt="Screen Shot 2020-09-24 at 9 46 06 PM" src="https://user-images.githubusercontent.com/778171/94227603-b7868600-feaf-11ea-9e9c-588191cde9f3.png">
